app/controllers/use_case_controller.rb

```diff
@@ -66,6 +60,18 @@ def attribute_changes?
     !@model_instance.changed_attributes.blank?
   end
 
+  def build_model_instance
+    if params[:action] == 'new'
+      filtered_params = {}
+    else
+      filtered_params = self.permitted_params
+    end
+    @model_instance = self.model_class.new(filtered_params)
+
+    # TODO unify with how the factories create model instances
```

Comment:
ah! did not realize that. 

sorry to badger you on this front. It is out of selfish desire to be able to help with small refactors. Currently feeling like I am still new enough to this project that inline todos are not actionable for me (don't have enough context yet)